lockfile/src/python.rs

```diff
@@ -104,7 +104,9 @@ impl Parse for Poetry {
         let mut lock: PoetryLock = toml::from_str(data)?;
 
         // Warn if the version of this lockfile might not be supported.
-        if !lock.metadata.lock_version.starts_with("1.") {
+        if !lock.metadata.lock_version.starts_with("1.")
+            && !lock.metadata.lock_version.starts_with("2.")
+        {
             log::debug!(
                 "Expected poetry lockfile version ^1.0.0, found {}.",
```

Comment:
```suggestion
                "Expected poetry lockfile version ^1.0.0 or ^2.0.0, found {}.",
```

Reply:
Honestly please feel free to just mark reviews as requesting changes if there's **anything** you're not sure about. If I really disagree we can figure that out later, I don't need the approval unless you think everything's perfect.

Otherwise it just risks accidentally merging things because there's a green checkmark and everything seems good to go when you miss a conversation somewhere.